libsrc/db/DBManager.cpp

```diff
@@ -8,6 +11,11 @@
 #include <QUuid>
 #include <QDir>
 
+#ifdef ENABLE_AMLOGIC
```

Comment:
Please change to #if ENABLE_AMLOGIC.
The variable ENABLE_AMLOGIC exists in any case and should only be checked for its value.

Reply:
Looks like this is not working without a compare.
So how to handle this? There are some more #ifdef like I used:
https://github.com/hyperion-project/hyperion.ng/blob/master/libsrc/grabber/framebuffer/FramebufferFrameGrabber.cpp#L49